schema.py

```diff
@@ -0,0 +1,8 @@
+from flask_sqlalchemy import SQLAlchemy
+
+db = SQLAlchemy()
+
+class User(db.Model):
+    __tablename__ = "USERS"
```

Comment:
There should be a column for timestamp because when u are retrieving to display in admin page you should display the latest records first

Reply:
I have created the timestamp column with the name 'created_at' Mam.